### PR TITLE
Update widgets.po

### PR DESCRIPTION
**Pull request by malikrohit16**
File was partly untranslated, which is now completely translated to German.

### CODE REVIEW

**Comment by jensens** [comment]
Needs review, some refinement needed. I will do tomorrow if nobody else is faster

**Comment by malikrohit16** [comment]
Ok that will be great!

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 1]
```diff
@@ -134,11 +134,11 @@ msgstr "Bundle von dem dies abhängig ist"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Bundles"
-msgstr ""
+msgstr "Bündel"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: This is a technical term and should stay english.

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 2]
```diff
@@ -134,11 +134,11 @@ msgstr "Bundle von dem dies abhängig ist"
...
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "CSS/LESS"
-msgstr ""
+msgstr "CSS/LESS"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: if the translated value is the same as the default it should not be translated but stay empty

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 3]
```diff
@@ -154,7 +154,7 @@ msgstr "Datei wählen"
 
 #: ./patterns/pickadate/pattern.js
 msgid "Clear"
-msgstr ""
+msgstr "klar"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: That makes no sense. It is about clearing/removing the current value.

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 4]
```diff
@@ -358,7 +358,7 @@ msgstr "Abgelaufen"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Export"
-msgstr ""
+msgstr "Export"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: Same as default: leave empty

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 5]
```diff
@@ -422,11 +422,11 @@ msgstr "Bild"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Init"
-msgstr ""
+msgstr "Init"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: Same as default: leave empty

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 6]
```diff
@@ -422,11 +422,11 @@ msgstr "Bild"
...
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Init instruction for requirejs shim"
-msgstr ""
+msgstr "Init-Anweisung für die Shim-Anforderung"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: requirejs is a js-library and cannot be translates as requirement.

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 7]
```diff
@@ -446,15 +446,15 @@ msgstr "Intern"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Internet Explorer conditional comment"
-msgstr ""
+msgstr "Bedingter Kommentar zum Internet Explorer"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: bad translation. This refers to a version-constraining comment for IE. like `<!--[if lt IE 7 ]>`

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 8]
```diff
@@ -446,15 +446,15 @@ msgstr "Intern"
...
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "JS"
-msgstr ""
+msgstr "JS"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: Same as default: leave empty

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 9]
```diff
@@ -446,15 +446,15 @@ msgstr "Intern"
...
 
 #: ./patterns/thememapper/js/lessbuilderview.js
 msgid "LESS Builder"
-msgstr ""
+msgstr "LESS Builder"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: Same as default: leave empty

File: plone/app/locales/locales/de/LC_MESSAGES/widgets.po [hunk 10]
```diff
@@ -574,7 +574,7 @@ msgstr "Aktualisieren"
 
 #: ./patterns/resourceregistry/pattern.js
 msgid "Registry"
-msgstr ""
+msgstr "Registrierung"
```
Comment (pbauer) [inline plone/app/locales/locales/de/LC_MESSAGES/widgets.po]: The Registry is the name of the component to configure ressources (i.e. the first tab in http://demo.plone.de/@@resourceregistry-controlpanel), not to regierster Users. So I'd either use the default or use "Ressourcen"

**Comment by malikrohit16** [comment]
@pbauer please review.

**Comment by malikrohit16** [comment]
hello @pbauer  I am Rohit a GSOC aspirant. I would like to work on user as content idea. I saw that you are mentor for this project so I wanted some pointers on the idea and get to know the idea in more depth. I would like to know how you envision the project like this working. It will be a great help to me.
Thanks

**Comment by jensens** [comment]
@malikrohit16 this (a PR about german translation) is probably the wrong place to ask the question. Best look at https://community.plone.org GSOC articles and ask there.

**Pull request merged**